backend/models/group.ts

```diff
@@ -0,0 +1,25 @@
+import { ObjectId } from '@modules/interfaces';
+import * as Mongoose  from 'mongoose';
+import {
+    GroupMethods
+} from        '@models/declarations/group';
+export * from '@models/declarations/group';
+
+export const Schema = new Mongoose.Schema({
+    name_grop:           { type: String,   required: true    },
+    group_id:            { type: Number,   required: true    },
+    num_of_participants: { type: Number,   default: 0        },
```

Comment:
Please, specify `required: true/false` option **for each** field, I will add this item to code conventions.
`default` option doesn't have any implications over `required: false/true`, as an example we define `init_date` as `required: true, default: Date.now`